integration-tests/pom.xml

```diff
@@ -309,39 +309,34 @@
             <properties>
                 <skipITs>false</skipITs>
                 <includes-failsafe>
-                   **/ItCoherenceTests,
+                   <!--**/ItExternalLbTunneling,-->
```

Comment:
you can delete the comment out lines and it is better to put the It* classes in alphabetical order.

Reply:
Deleted the commented tests and sorted it in alphabetical order.